src/tests/jsonLogic.test.js

```diff
@@ -210,4 +211,16 @@ describe('cross-value validations', () => {
       expect(handleValidation({ field_a: 4, field_b: 2 }).formErrors).toEqual(undefined);
     });
   });
+
+  describe('Derive values', () => {
+    it('field_b is field_a * 2', () => {
+      const { fields } = createHeadlessForm(schemaWithComputedAttributes, {
+        strictInputType: false,
+        initialValues: { field_a: 2 },
+      });
+      const fieldB = fields.find((i) => i.name === 'field_b');
+      expect(fieldB.default).toEqual(4);
+      expect(fieldB.value).toEqual(4);
```

Comment:
todo/bug: I didn't remember this before, but I'm concerned with this new `value` key because it might conflict with our internal codebase (`composeField*` fns) where value is already used for other legacy things.
Hopefully, I'm wrong, but could you please open a PR with a demo to test that it integrates well out of the box? (related [linear issue](https://linear.app/remote/issue/RMT-18))

Reply:
On it.

Reply:
Internal MR opened

Reply:
will review it later today!